addons/mail/static/src/core/common/message_reaction_list.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<template xml:space="preserve">
+    <t t-name="mail.messageReactionList">
+        <Dropdown state="preview" position="'bottom-start'" menuClass="'bg-view border-0 p-0 mt-1 overflow-visible rounded-3'" arrow="true" manual="true">
```

Comment:
`cursor-pointer` on `menuClass`, so we know this is clickable. This is actually better than underline if we had to choose only one. I removed the underline because this would require partial styling of a sentence, which is quite challenging to make it work in all supported languages, so I prefer to remove it to not have to deal with.

Reply:
I also removed `arrow="true"`, because this didn't look good